src/components/Region.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Region = (props) => {
+  const { data } = props;
```

Comment:
**Optional**

- [ ] You can save one line of code by directly destructuring the `data` properties inside the function parameter like so:

```
const Region = ({ data }) => {
 other codes..
 .....
}
```